src/components/CommentsBox.js

```diff
@@ -24,16 +24,15 @@ const CommentsBoxContent = (props) => {
             {/* {TODO: Put in conditionals and variables for the new comment helper} */}
             <div className="w-full flex justify-between items-baseline font-sans mb-10">
                 <h2 className="font-bold text-[2.8rem] tracking-tight dark:text-[rgba(255,255,255,0.85)]">
-                    {/* This will truncate to "Discussion" on mobile screens to save space */}
-                    <span className="hidden sm:inline">Member </span><span className="capitalize sm:normal-case">discussion</span>
+                    {title ? title : <><span className="hidden sm:inline">Member </span><span className="capitalize sm:normal-case">discussion</span></>}
                 </h2>
                 <div className="text-neutral-400 text-[1.6rem] font-medium">{commentsCount} comments</div>
             </div>
             <Pagination />
-            <div className={!pagination ? 'mt-4' : ''}>
+            {count ? <div className={!pagination ? 'mt-4' : ''}>
```

Comment:
@allouis Do we inject the default title and count as data attributes on script tag if someone uses the vanilla helper - `{{comments}}` ?

Reply:
https://github.com/TryGhost/Ghost/pull/15112 These are the changes to the helper

We use `null` instead of the default title, so that we can do the mobile friendly stuff as discussed in the slack thread